static/styles/portico/markdown.css

```diff
@@ -179,7 +179,7 @@
     }
 
     a {
-        color: hsl(176, 46%, 41%);
+        color: hsl(176deg 27% 32%);
```

Comment:
What's the story with changing the formatting for how we encode colors?  If it makes sense to do, we should do it in a separate commit from this change in any case.

Reply:
Are you talking about the difference between `176` and `176deg`, the _deg_ which I added?
When I changed the color using VScode, it automatically changed it to that.

Reply:
Both the added `deg` and the deleted commas are invalid until very recently ([CSS Colors level 4](https://www.w3.org/TR/css-color-4/)). We should stick with the existing style for now.

Reply:
Yeah, you should turn off the responsible VSCode plugin and undo changes like this -- autoreformatting that codebase to a different style from that codebase's preferred style is never something you want your editor to do :).

Let me know when you've cleaned this up.